src/lib.rs

```diff
@@ -24,13 +25,14 @@ mod source;
 mod store;
 
 pub use crate::annotations::{
-    Annotation, Associative, Combine, ErasedAnnotation, VoidAnnotation,
+    Annotation, Associative, Combine, ErasedAnnotation, Void,
```

Comment:
Just curious why the change of name here. I thought it made more consistent especially because we have now `ErasedAnnotation`. Otherwise we might have `Erased` and `Void` but then to avoid clashing with the `Erased` type we might need `annotations::Erased` and `annotations::Void`.

I mean, I' m fine either way, I just want to be sure that annotations have the same naming convention so it's clear at first sight what we're dealing with.

Reply:
added issue #43 as a low-prio improvement.